sources/ClangSharp.PInvokeGenerator/PInvokeGenerator.VisitDecl.cs

```diff
@@ -1479,6 +1479,7 @@ private void VisitRecordDecl(RecordDecl recordDecl)
                     Size32 = size32,
                     Size64 = size64,
                     Pack = alignment < maxAlignm ? alignment : 0,
```

Comment:
Why not just pass the value into `Pack` and have it fallback to `((alignment < maxAlignm) ? alignment : 0)`?

Reply:
`PackOverride` is a string, so that it can be set to the name of a constant. If I used `Pack` it'd need to have its type changed to accommodate that.

Reply:
Right, that seems fine overall. Just use `null` or `empty string` to denote `0` instead.

Reply:
Addressed in most recent commit.